libpod/kube.go

```diff
@@ -1390,17 +1390,17 @@ func removeUnderscores(s string) string {
 // and returns them as kube annotations
 func getAutoUpdateAnnotations(ctrName string, ctrLabels map[string]string, useLongAnnotations bool) map[string]string {
 	autoUpdateLabel := "io.containers.autoupdate"
-	annotations := make(map[string]string)
+	a := make(map[string]string)
```

Comment:
I prefer the original here. Dealer's choice, though.

Reply:
Except for iterators, I'm not at all a fan of single-character variable names.  When you're looking for the variable `a` in a 1400+ piece of code, good luck to the support engineer three years hence.